linuxdeploy-plugin-gtk.sh

```diff
@@ -315,6 +304,7 @@ PATCH_ARRAY=(
 for directory in "${PATCH_ARRAY[@]}"; do
     while IFS= read -r -d '' file; do
         # shellcheck disable=SC2016
-        patchelf --set-rpath '$ORIGIN/../../../..' "$APPDIR/$file"
+        patchelf --set-rpath '$ORIGIN' "$APPDIR/$file"
```

Comment:
Not sure if still required. linuxdeploy.AppImage already to this automatically if I am correct.

Reply:
I think the main issue with this line is that it forcefully overwrites the existing rpath. I think linuxdeploy now just appends to the rpath, in case it exists.

Reply:
Ok, I removed `patchelf` dependency in 07ebdecaff325b6bcabe28436aaaf7294c2dc327.